diff --git a/node_modules/nx/bin/init-local.js b/node_modules/nx/bin/init-local.js
index 3cbb798..b250eb4 100644
--- a/node_modules/nx/bin/init-local.js
+++ b/node_modules/nx/bin/init-local.js
@@ -11,6 +11,9 @@ const Mod = require("module");
  * @param workspace Relevant local workspace properties
  */
 function initLocal(workspace) {
+    console.log('>>> Nx', {
+      argv: process.argv,
+    });
     process.env.NX_CLI_SET = 'true';
     try {
         perf_hooks_1.performance.mark('init-local');
diff --git a/node_modules/nx/bin/nx.js b/node_modules/nx/bin/nx.js
index d0ee823..3ccfdaf 100755
--- a/node_modules/nx/bin/nx.js
+++ b/node_modules/nx/bin/nx.js
@@ -18,6 +18,10 @@ const perf_hooks_1 = require("perf_hooks");
 const workspace_context_1 = require("../src/utils/workspace-context");
 const client_1 = require("../src/daemon/client/client");
 function main() {
+  console.log('>>>> Nx:', {
+    'argv': process.argv,
+    cwd: process.cwd(),
+  });
     if (process.argv[2] !== 'report' &&
         process.argv[2] !== '--version' &&
         process.argv[2] !== '--help') {
